OJP/OJP_JourneySupport.xsd

```diff
@@ -357,6 +367,9 @@
 		</xs:annotation>
 		<xs:sequence>
 			<xs:group ref="ServiceTimeGroup"/>
+			<xs:element ref="siri:ArrivalFormationAssignment" minOccurs="0"/>
+			<!-- comes with SIRI 2.1 -->
+			<!-- TODO the quays should be moved here, too -->
```

Comment:
Is this comment needed/useful?

Reply:
I think for following up (when updating to SIRI 2.1 it is)